### DISCUSSION TITLE
Loading signs from anvil world

### DISCUSSION THREAD

**Discussion by Rqver** [Minestom/Minestom#657] [Help] [answered]
Hi,

How would I load the contents & colors of signs from an anvil world?

When I load in the world the signs are there but they have nothing on them.

Thanks

**Comment by emortaldev** [comment, marked as answer]
You will need a block handler like [this](https://github.com/EmortalMC/Immortal/blob/main/src/main/kotlin/dev/emortal/immortal/blockhandler/SignHandler.kt) (easily translated to Java), which can then be registered with `MinecraftServer.getBlockManager().registerHandler("minecraft:sign", () -> new SignHandler())`

**Reply by Rqver** [reply]
Ty :)
